MIP101/MIP101.md

```diff
@@ -124,19 +122,19 @@ Alignment Conservers are external entities that play a fundamental role in facil
 
 Voter Committees are groups of MKR holders that publicly coordinate to analyze the best voting behavior based on a particular strategy, and publicly coordinate political positioning with other Voter Committees and voting blocs to reach compromises and middle grounds that minimize bad decisions. The base form of Voter Committees are the Permissionless Voter Committees which is a very simple designation for any kind of organized voting group.
 
-AVCs are standardized Voter Committees made up of Alignment Conservers that hold MKR, and publicly coordinate to analyze the best voting behavior based on a particular strategy, and publicly coordinate political positioning with other Voter Committees and voting blocs to reach compromises and middle grounds that minimize bad decisions. They are subject to specific requirements enforced by the Governance Scope, and receive various benefits, resources and support from the Support Scope.
+<abbr title="Aligned Voter Committees">AVCs</abbr> are standardized Voter Committees made up of Alignment Conservers that hold MKR, and publicly coordinate to analyze the best voting behavior based on a particular strategy, and publicly coordinate political positioning with other Voter Committees and voting blocs to reach compromises and middle grounds that minimize bad decisions. They are subject to specific requirements enforced by the Governance Scope, and receive various benefits, resources and support from the Support Scope.
```

Comment:
I'm not really clear what the purpose of the added HTML is here, could you explain please?